### ISSUE TITLE
Making the website responsive

### ISSUE THREAD

**Issue by Bhum-ika** [Bitbox-Connect/Bitbox#3]
Kindly assign this issue to me under hactoberfest, gssoc-ext and relevant tags so I can start working on its responsiveness

**Comment by Anuj3553** [comment]
> Kindly assign this issue to me under hactoberfest, gssoc-ext and relevant tags so I can start working on its responsiveness

Hey Bhumika,

I have given the permission to clone the project. Check it out and accept the invitation in your profile.

**Comment by Bhum-ika** [comment]
Accepted.
KIndly assign the required badges

**Comment by Anuj3553** [comment]
> Accepted. KIndly assign the required badges

First, don't worry about the badge. We will assign it to you after you complete the task. Once the task is completed, we will give you the badge.

**Comment by Anuj3553** [comment]
If you want to start working, we will create a new branch for you so you can work seamlessly. We will check your branch to ensure it is compatible, and then we will push your code.

**Comment by Bhum-ika** [comment]
Okay, sure

**Comment by Anuj3553** [comment]
> Okay, sure

What kind of work would you like to do? Can you give me some ideas? What is your area of expertise in the tech stack?

**Comment by Bhum-ika** [comment]
As I mentioned previously, I will ensure that the website is responsive on all devices. My expertise lies in React and Tailwind CSS.

**Comment by Anuj3553** [comment]
> As I mentioned previously, I will ensure that the website is responsive on all devices. My expertise lies in React and Tailwind CSS.

Okay bhumika, If you have any questions or additional information, free feel to add them here. Your contributions are highly appreciated 😊.

**Comment by Bhum-ika** [comment]
Hi Anuj, your project already has Tailwind installed, so why is it coded in CSS? I can convert the entire CSS into Tailwind, which would be more convenient.
Please let me know.

**Comment by Anuj3553** [comment]
Hey @Bhum-ika,

Thanks for pointing that out! Feel free to go ahead and convert the CSS into Tailwind. It would definitely make things more streamlined and convenient. Appreciate your help!

**Comment by Bhum-ika** [comment]
Okay, so what i'll do is i am raising an issue for conversion and this issue will be for making it responsive.
Is that okay?

**Comment by Anuj3553** [comment]
I'll edit the issue title, so this issue is currently aligned with your requested requirements.

**Comment by Bhum-ika** [comment]
Okay

**Comment by Bhum-ika** [comment]
Hi @Anuj3553 

![image](https://github.com/user-attachments/assets/236ab419-7401-4280-8269-064b5d5f7068)


It is running like this locally. Do you know what might be the issue

**Comment by Anuj3553** [comment]
Do you mean localhost://PORT

**Comment by Bhum-ika** [comment]
Yeah

**Comment by Anuj3553** [comment]
Yes it is running in your local system and pre production is always will be on locally.

**Comment by Bhum-ika** [comment]
I am trying to convey that only this section is visible, without any header, footer, or other components.

**Comment by Anuj3553** [comment]
> I am trying to convey that only this section is visible, without any header, footer, or other components. How am I suppose to see other components

I think it is a mistake of someone contributor's.

**Comment by Bhum-ika** [comment]
Yes, please check

**Comment by Anuj3553** [comment]
Sorry, unfortunately i am busy right now in my company. Can you fix this?

**Comment by Bhum-ika** [comment]
I'll see into it

**Comment by Anuj3553** [comment]
The mentors are also doing nothing. I have to handle all the things.

**Comment by Bhum-ika** [comment]
Yeah, I know
Don't worry if this issue persists I'll fix it

**Comment by Anuj3553** [comment]
You can raise a new issue for this. So for this it is counting marks separately.

**Comment by Bhum-ika** [comment]
Sure, thank you

**Comment by Bhum-ika** [comment]
@Anuj3553 can you brief the steps for running the backend of this project, I have connected MongoDB, but from there, it got stuck

**Comment by Anuj3553** [comment]
Okk, share the error to me.

**Comment by Anuj3553** [comment]
Check the mongodb string is correct or not. If there is problem check to locally by using mongodb compass.

**Comment by Bhum-ika** [comment]
If you'll just run the frontend, you'll encounter that issue too

**Comment by Bhum-ika** [comment]
It's working but i am not able to run it, only frontend is  running currently on my laptop and seeing the code, seems it will work fine when client and server both are running. 
Could you please check once and if that's the issue only then kindly brief the steps for running server after connecting mongoDB

**Comment by Anuj3553** [comment]
> It's working but i am not able to run it, only frontend is running currently on my laptop and seeing the code, seems it will work fine when client and server both are running. Could you please check once and if that's the issue only then kindly brief the steps for running server after connecting mongoDB

Hyy @Bhum-ika,

Step1 -> Open first terminal and type -> cd client, npm i and then npm run dev
Step1 -> Open second terminal and type -> cd server, npm i and then nodemon index.js

**Comment by Bhum-ika** [comment]
Hey, aren't you on discord. I am getting some errors after running nodemon command

**Comment by Anuj3553** [comment]
Yes I'm in discord. Here's my id - anujv3553. If you go to my profile i provided social link to easily connect with me.

**Comment by Bhum-ika** [comment]
Please check discord.
